src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -468,11 +468,11 @@
             </trans-unit>
             <trans-unit id="120">
                 <source>This value is not a valid slug.</source>
-                <target state="needs-review-translation">Šī vērtība nav derīgs slug.</target>
+                <target>Šī vērtība nav derīgs URL fragments.</target>
```

Comment:
A URL fragment is not the same as a slug. Why not keep slug here?

Reply:
Agree, reverted. We have a word for `path` but not for a `slug`: https://lv.wikipedia.org/wiki/Vienotais_resursu_vietr%C4%81dis